ui/src/overridden/backoffice/Items/ItemCirculationShelf.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Grid, Embed, Button, Icon } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { shelfLink } from "../../utils";
+
+export const ItemCirculationShelf = ({ metadata }) => {
+  return (
+    <Grid.Column width={6}>
+      <>
+        <Embed
+          active
+          url={shelfLink(metadata.shelf, true)}
+          style={{
```

Comment:
Could you move the styling to `site.overridable`? Inline styling is generally among bad practices